src/ui/deleteAccount/ViewDeleteAccountRequest.qml

```diff
@@ -82,6 +82,7 @@ VPNInAppAuthenticationBase {
             // Delete account
             text: VPNl18n.DeleteAccountButtonLabel
             onClicked: if (viewDeleteAccount.allowAccountDeletion) {
+                Sample.deleteAccountClicked.record();
```

Comment:
I would like to move this in auth-in-app so that all the auth-in-app signals are in 1 single place.